packages/block-library/src/footnotes/index.php

```diff
@@ -243,7 +243,7 @@ function _wp_rest_api_autosave_meta( $autosave ) {
 		return;
 	}
 
-	update_post_meta( $id, 'footnotes', $body['meta']['footnotes'] );
+	update_post_meta( $id, 'footnotes', wp_slash( $body['meta']['footnotes'] ) );
```

Comment:
Auto saving still triggers the `count()` error for me, but not when I add `wp_slash()` to the update in `wp_save_footnotes_meta`, e.g., 


```php
update_metadata( 'post', $revision_id, 'footnotes', wp_slash( $footnotes ) );
```

I guess that's being called as well?